### DISCUSSION TITLE
Set always day

### DISCUSSION THREAD

**Discussion by espen4001** [itzg/docker-minecraft-bedrock-server#203] [Q&A]
How do I set the "always day" switch in the server? I tried to enable cheats and writing the command /gamerule dodaylightcycle false, but does not work.
I managed to set to daylight, but not to stop the cycle.

Is it possible to set it in the docker-compose-file?

**Comment by itzg** [comment]
Game rules can't be set via server properties and therefore can't be set via the container environment variables. This tool looks like it might be helpful though to adjust the word data offline

https://docs.microsoft.com/en-us/minecraft/creator/documents/chunkerworldsettings
